frontend/src/features/research/activities/list/ActivityResults/AddActivityForm.tsx

```diff
@@ -0,0 +1,87 @@
+import { PlusButton } from 'components/common/buttons';
+import { Select, SelectOption } from 'components/common/form';
+import { Formik } from 'formik';
+import { useApiResearchFile } from 'hooks/pims-api/useApiResearchFile';
+import useIsMounted from 'hooks/useIsMounted';
+import { Api_Activity, Api_ActivityTemplate } from 'models/api/Activity';
+import React, { useCallback, useState } from 'react';
+import { Col, Row } from 'react-bootstrap';
+
+export interface IAddActivityFormProps {
+  onAddActivity: (activity: Api_Activity) => void;
+}
+
+export interface IAddActivity {
+  activityTypeId?: string;
+}
+
+export function AddActivityForm(props: IAddActivityFormProps) {
+  const { onAddActivity } = props;
+  const isMounted = useIsMounted();
+  const { getActivityTemplates } = useApiResearchFile();
+  const [templateTypes, setTemplateTypes] = useState<SelectOption[]>([]);
+
+  const fetchActivityTemplates = useCallback(async () => {
```

Comment:
not a fan of having the form do api calls. Could this information be provided to the form?

Reply:
moved to parent list view component